proxy/certs/certs.go

```diff
@@ -304,8 +342,55 @@ func (s *RemoteCertSource) findIPAddr(data *sqladmin.ConnectSettings, instance s
 func (s *RemoteCertSource) Remote(instance string) (cert *x509.Certificate, addr, name, version string, err error) {
 	p, region, n := util.SplitName(instance)
 	regionName := fmt.Sprintf("%s~%s", region, n)
-	req := s.serv.Connect.Get(p, regionName)
+	if s.UseSslCerts {
+		req := s.serv.Instances.Get(p, regionName)
+		var data *sqladmin.DatabaseInstance
+		err = backoffAPIRetry("get instance", instance, func() error {
+			data, err = req.Do()
+			return err
+		})
+		if err != nil {
+			return nil, "", "", "", err
+		}
+
```

Comment:
considering add the logging.Verbose  message like "get SSL EphemeralCert successful" in s.UseSslCerts section, while "get Connect SSL EphemeralCert successful" in connect APIs. This will help to debug which API is called.